core/test/audits/dobetterweb/uses-http2-test.js

```diff
@@ -5,26 +5,55 @@
  */
 
 import UsesHTTP2Audit from '../../../audits/dobetterweb/uses-http2.js';
-import {NetworkRecords} from '../../../computed/network-records.js';
 import {networkRecordsToDevtoolsLog} from '../../network-records-to-devtools-log.js';
-import {getURLArtifactFromDevtoolsLog, readJson} from '../../test-utils.js';
+import {createTestTrace} from '../../create-test-trace.js';
 
-const trace = readJson('../../fixtures/traces/progressive-app-m60.json', import.meta);
-const devtoolsLog = readJson('../../fixtures/traces/progressive-app-m60.devtools.log.json', import.meta);
+const mainResources = [{
```

Comment:
Why do we need 3 main resource requests with the same URL in the tests?

Reply:
this was to cover the url limit in https://github.com/GoogleChrome/lighthouse/pull/15320/files#diff-ccdb44e5509ddacd7a1c43834f27f2e23caa6ce9d0bc03c63bfec031966f9c46R224, but maybe there's a better way to go about this?

Reply:
Gotcha, just for understand could we do:
- Rename `mainResources` to `baseResources` or something
- Update the URLs of each resource so they are not the same
- Make them the first resources in the network records list rather than the last